ocfweb/docs/docs/staff/backend/rt.md

```diff
@@ -0,0 +1,57 @@
+[[!meta title="Request Tracker"]]
+
+[**Request Tracker**](https://rt.ocf.berkeley.edu/) is the ticketing system
+used by the OCF. It is the main way of keeping track of OCF-related activity.
+Tickets are currently made either by staff to propose things to talk about
+or by emails from the Hostmaster or Help mailing lists.
+
+## Queues
+Tickets are assigned to queues, or organized boards. Manually-created
+tickets are found under:
+- *bod* for meeting topics
+- *bureaucracy* for officer-related issues
+- *operations* for Operations Strategist work (opstaff)
+- *projects* for long-term activities
+- *techtalks* for Tech Talk ideas and organization
+- *todo* for shorter-term items
+
+Other tickets are created automatically from emails:
+- *devnull* for devnull@ emails
+- *help* for questions from users (help@)
+- *hostmaster* for requests for virtual hosting ([form](https://www.ocf.berkeley.edu/account/vhost/))
+- *security* for vulnerability detections from ISP
+
+All of these queues are also associated with a mailing alias.
+
+## Tickets
+### Comment vs Reply
+Much like the issues between Reply and Reply-All, the difference between Comment and Reply has led
+to some mishaps. In the RT interface, *Reply* directly communicates with the poster, so look for the
+last communication with the ticket opener. *Comment* doesn't directly communicate and is generally for
+internal discussion. This can also be done through email, as RT defaults the reply-to field with the
+queue mailing lists. Be careful here though: to comment through email, send the email to {queue}-comment
+(i.e. help@ vs help-comment@). Also make sure that your reply does not include any of the comments, as in
+make sure the trimmed comment is all the information you want released.
+
+### Creation
+They can be manually created through the *New Ticket in* button on the top right of the page. If doing
+so to communicate to people outside of the OCF, add their email to the requestors field of the ticket.
```

Comment:
when creating tickets in order to communicate with outsiders, it's necessary to:

1. create a new ticket in the web UI, specifying them as the requestor. leave the body of the ticket entirely blank
2. reply to the newly-created ticket, including your initial email in the body

this is necessary because the first step doesn't generate an email to the requestor (but does email staff); the second step will email them

might be worth documenting this somewhere since it confuses people sometimes